Emmanuel-Nkurunziza/my-brand
# #4 Build out the composite landing page

#### What does this PR do?

•	It adds the navigation bar with buttons such as about me, my articles, my work, contact me, and My brand logo on the topmost frame of the landing page. It adds also the site owner picture, the welcoming note, and the allow-down to the about me frame of the landing page

•	It adds the professional history of the site owner to the about me frame of the landing page. It adds also the allow-down link to the blog post page

•	it adds a general description of the usefulness of articles written by the site owner on the blog post page. It adds also an invitation to read, like, and comment on the articles. It adds also an allow button guiding the visitor to the articles. Also, the navigation-allow-down to the following page of site owner projects is added.

•	it adds the 3 projects that the site owner participated in into the frame of my work on the Landing page. And an invitation to write to the site owner for questions or more information alongside the navigation-allow button points to the contact me page is also added.

•	it adds the form to be filled by visitors with their name/email/question or message to this contact me frame of the landing page. It adds also the allow-up button pointing on the upper frame of the landing page. The site-owner social media handles (GitHub, tweeter, google+, Facebook, and LinkedIn) are also added


#### Description of Task to be completed

•	upon click to the about me in the nav bar or the about me allow-down the visitor lands to the about me frame of the landing page
•	upon click to my articles in the nav bar or the allow-down found in about me frame of the landing page the visitor lands to the blog post frame of the landing page
•	upon click to my work in the nav bar or the allow-down found in the blog past frame of the landing page the visitor lands to the projects frame of the landing page
•	upon click to contact me in the nav bar or the allow-down found in the projects frame of the landing page the visitor lands to the contact me frame of the landing page


#### How should this be manually tested?

•	clone the repo and switch to the branch ft-landing-page-#4  
•	use the url https://emmanuel-nkurunziza.github.io/
•	you should successfully see the above-mentioned features of the index.html


#### Any background context you want to provide?

•	N/A


#### What are the relevant trello stories?

https://trello.com/c/NSjiUPvC/4-landing-page



#### Screenshots (if appropriate)


![image](https://user-images.githubusercontent.com/52543344/146799015-07bbf52a-976e-4bad-b53a-a55c1636a7e1.png)

![image](https://user-images.githubusercontent.com/52543344/146799090-a5bde1ec-e0ca-4c5d-8e9f-a3de16db8f7d.png)

![image](https://user-images.githubusercontent.com/52543344/146799188-8a90f814-38e7-47be-9bba-0b5b6845dc5e.png)

![image](https://user-images.githubusercontent.com/52543344/146799254-8baa1255-8a3a-444b-9b7c-7781231032a4.png)

![image](https://user-images.githubusercontent.com/52543344/146799343-d01c154f-0989-4179-b807-1ad06973fe70.png)
